five.txt

```diff
@@ -0,0 +1 @@
+five
```

Comment:
Instead of writing `five` maybe the number `5` is enough 🔥 🚀 
But anyways it works